tutorials/vr/vr_primer.rst

```diff
@@ -127,13 +127,15 @@ in the `GodotVR Repository <https://github.com/GodotVR>`_.
   :ref:`doc_developing_for_oculus_quest`.
 * `Godot OpenVR <https://github.com/GodotVR/godot_openvr>`_ (not to be confused with OpenXR)
   supports the OpenVR SDK used by Steam.
-* `Godot Oculus <https://github.com/GodotVR/godot_oculus>`_ supports the Oculus SDK
-  (desktop headsets only).
```

Comment:
I'm not sure it should be removed. It can be marked as deprecated though.

I'd put OpenXR as the first entry.

But mostly up to @BastiaanOlij to decide.

Reply:
To be honest, seeing the OpenXR plugin is now as far along as it is, it makes total sense to just name the OpenXR plugin here, and then have a note that for the time being you need the oculus mobile plugin to run on quest. 
Then maybe do a little more work to convert the tutorial to be fully OpenXR centered.

Seeing both Valve and Oculus seem to be on a path of moving fully to OpenXR it future proofs this tutorial